osf/management/commands/sync_datacite_doi_metadata.py

```diff
@@ -46,7 +53,15 @@ def add_arguments(self, parser):
             action='store_true',
             dest='dry_run',
         )
+        parser.add_argument(
+            '--batch_size',
+            '-b',
+            type=int,
+            default=0,
+            help='number of dois to create.',
+        )
 
     def handle(self, *args, **options):
         dry_run = options.get('dry_run')
-        sync_datacite_doi_metadata(dry_run=dry_run)
+        batch_size = options.get('batch_size', 0)
+        sync_datacite_doi_metadata(dry_run=dry_run, batch_size=batch_size)
```

Comment:
You have a default `batch_size` in the function signature that you're ignoring/replacing here. Why not use the same default in both places?

Reply:
still functionally overruling the function's default `batch_size` here, since `--batch_size` has a default of zero.